python/lammps/core.py

```diff
@@ -1299,6 +1308,75 @@ def gather_bonds(self):
 
   # -------------------------------------------------------------------------
 
+  def gather_angles(self):
+    """Retrieve global list of angles
+
+    This is a wrapper around the :cpp:func:`lammps_gather_angles`
+    function of the C-library interface.
+
+    This function returns a tuple with the number of angles and a
+    flat list of ctypes integer values with the angle type, angle atom1,
+    angle atom2, angle atom3 for each angle.
+
+    .. versionadded:: 28Jan2023
```

Comment:
The version number for versionadded/versionchanged should be "TBD". This applies to all similar tags in the remaining changes. The TBD will be replaced with the actual version in the last pull request before the release that also updates src/version.h